Resources/Locale/en-US/administration/commands/spawncharacter.ftl

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
This should either be a _DV folder, or have a DeltaV comment. Not sure if we did it this way for other commands, but that's how it _should_ be.

Reply:
Good catch. I'll put it in a _DV directory.